src/pages/common/Categories/Categories.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import "./style.css";
+import { ICategories } from "../../../interfaces/MovieInterfaces";
+import { filterMovie } from "../../../store/reducers/favouriteSlice";
+import { useAppDispatch } from "../../../hooks/useAppDispatch";
+
+const Categories = () => {
+  const dispatch = useAppDispatch();
+  const chooseCategory = (category: any) => {
+    dispatch(filterMovie(category));
+  };
+  const categories: ICategories[] = [
```

Comment:
Move out of component since it's static data and will not rerender if it would define outside a component. 
<img width="407" alt="image" src="https://user-images.githubusercontent.com/58829953/231276289-600e4e04-f833-4751-9c66-07219abc66c7.png">